script/ebooks

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+// Generate EPUB and MOBI ebooks
+
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+const fastmatter = require('fastmatter');
+const mkdirp = require('mkdirp');
+const slug = require('slug');
+const unified = require('unified');
+const parse = require('remark-parse');
+const stringify = require('remark-stringify');
+const visit = require('unist-util-visit');
+const download = require('download');
+const ebrew = require('ebrew');
+
+const processor = unified()
+  .use(parse)
+  .use(replaceAsides)
+  .use(replaceCheckboxes)
+  .use(fixImages)
+  .use(downloadImages)
+  .use(stringify);
```

Comment:
Yeah, in long term it has benefits like better control on the style and formatting, custom CSS, etc. But for an MVP Markdown is enough I think ;-)